### ISSUE TITLE
Peripheral Cyanosis

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7123]
**Preferred term label:**
peripheral cyanosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Bluish discoloration of the distal extremities (hands, fingertips, toes), and can sometimes involve circumoral and periorbital areas. Mucous membranes are generally not involved. [PMID: 32644593]

**Parent term (use hpo.jax.org/app)**
HP:0001063 - Acrocyanosis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Database cross reference**
[SNOWMEDCT_US:95442007](https://phinvads.cdc.gov/vads/ViewCodeSystemConcept.action?oid=2.16.840.1.113883.6.96&code=95442007)

**Comment by pnrobinson** [comment]
I think peripheral cyanosis should be a parent for acro and facial cyanosis.
Adding this term.
